source/_components/cover.template.markdown

```diff
@@ -33,7 +34,12 @@ cover:
           service: script.close_garage_door
         stop_cover:
           service: script.stop_garage_door
+        availability_template: >-
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.